tools/kochdocs.nim

```diff
@@ -49,7 +50,8 @@ template inFold*(desc, body) =
   if existsEnv("TRAVIS"):
     echo "travis_fold:end:" & desc.replace(" ", "_")
 
-proc execFold*(desc, cmd: string, errorcode: int = QuitFailure, additionalPath = "") =
+proc execFold*(desc, cmd: string, errorcode: int = QuitFailure,
```

Comment:
see https://nim-lang.github.io/Nim/contributing.html

> Do not mix pure formatting changes (e.g. whitespace changes, nimpretty) or automated changes (e.g. nimfix) with other code changes: these should be in separate commits (and the merge on github should not squash these into 1).

Reply:
My prettyfier auto formatted on save. Sorry.

Reply:
you probably should disable it when writing PR's; otherwise everytime you make a tiny edit in a file you'll drag along unrelated changes